packages/website/pages/files.js

```diff
@@ -44,6 +46,9 @@ export default function Files({ user }) {
   /** @type {[string, { before: string, limit: number }]} */
   const queryKey = ['get-nfts', queryParams]
 
+  let isDev
+  if (!!globalThis.window && location.host === 'localhost:4000') isDev = true
```

Comment:
Comment:
Is this how we find the environment of this app? I thought there should be an env variable set somewhere that would make this much easier.

Reply:
This is a request for more info, can be solved another time.